.github/actions/setup-terraform/action.yaml

```diff
@@ -11,7 +11,7 @@ runs:
   steps:
     - name: Set Terraform version
       shell: bash
-      working-directory: .infrastructure
+      working-directory: apps/infrastructure/src
```

Comment:
*optional*: Is there a way to retrieve it from the caller to prevent the repetition of `apps/infrastructure/src`?

Reply:
I'm going to try. If it's going to work, then the CI will not fail